app/src/main/java/org/fossasia/pslab/fragment/FAQFragment.java

```diff
@@ -0,0 +1,151 @@
+package org.fossasia.pslab.fragment;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseExpandableListAdapter;
+import android.widget.ExpandableListView;
+import android.widget.TextView;
+
+import org.fossasia.pslab.R;
+
+public class FAQFragment extends Fragment {
```

Comment:
PR looks good. This file needs reformatting. Also use variable names related to context rather than general names.

Reply:
Ya I'll change the names of the variables to more relevant ones.